flocx_market/objects/offer_contract_relationship.py

```diff
@@ -16,39 +16,35 @@ class OfferContractRelationship(base.FLOCXMarketObject):
     }
 
     @classmethod
-    def get(cls, context, contract_id, marketplace_offer_id):
-        if (contract_id is None) and (marketplace_offer_id is None):
+    def get(cls, context, ocr_id):
+        o = db.offer_contract_relationship_get(context, ocr_id)
+        if o is None:
             return None
         else:
-            o = db.offer_contract_relationship_get(
-                contract_id=contract_id,
-                marketplace_offer_id=marketplace_offer_id,
-                context=context)
-            if o is None:
-                return None
-            else:
-                return cls._from_db_object(cls(), o)
+            return cls._from_db_object(cls(), o)
 
     def destroy(self, context):
         db.offer_contract_relationship_destroy(
-            contract_id=self.contract_id,
-            marketplace_offer_id=self.marketplace_offer_id,
-            context=context)
+            context,
+            self.offer_contract_relationship_id)
         return True
 
     @classmethod
-    def get_all(cls, context):
-        all_ocrs = db.offer_contract_relationship_get_all(context)
-        return cls._from_db_object_list(all_ocrs)
+    def get_all(cls, context, filters=None):
+
+        o = db.offer_contract_relationship_get_all(context, filters)
+        if o is None:
+            return None
```

Comment:
Do we need this? `o` would be an empty list, which would be handled fine below, right?